src/pages/beta/[viewType]/curriculum/index.tsx

```diff
@@ -59,96 +72,102 @@ const CurriculumHomePage: NextPage<CurriculumHomePageProps> = (props) => {
-              </GridArea>
-              <GridArea $colSpan={[12, 4]}>
-                <Card
-                  $background="teachersPastelBlue"
-                  $pa={[24]}
-                  $height={[96]}
+          <Flex
+            $flexDirection={["column", "row"]}
+            $justifyContent={"space-between"}
+            $alignItems={"center"}
+            $gap={[24, 80, 120]}
+            $mb={80}
+            $pl={[16, 0]}
+            $pr={[16, 0]}
+            $mt={80}
+          >
+            <Box>
+              <Cover
+                $width={[320, 340, 480]}
+                $height={[300, 320, 450]}
+                $position={"relative"}
+              >
+                <Illustration
+                  noCrop
+                  sizes={getSizes([500, 800])}
+                  slug="curriculum-approach"
+                  $objectFit="contain"
+                  $objectPosition={"center"}
+                  fill
+                />
+              </Cover>
+            </Box>
+
+            <Box $height={"100%"}>
+              <Heading tag="h2" $font={["heading-5", "heading-4"]} $mt={12}>
+                Our approach to curriculum
+              </Heading>
+              <P $mv={[16, 24]} $font={"body-1"}>
+                Every school’s approach to curriculum design is different.
+                Whether you are starting from scratch or refreshing your
+                existing curricula, it can be helpful to see examples to inspire
+                your own. Explore our curriculum sequences and learn about the
+                six curriculum principles that guide our lesson and curriculum
+                design. See how we have:
+              </P>
+              <UL $mb={24} $font={"list-item-1"}>
+                <LI>
+                  Sequenced subject units across year groups, including
+                  alternative sequences
+                </LI>
+                <LI>Built coherence through the curricula, using threads</LI>
+                <LI>
+                  Summarised the core content pupils will learn, in our
+                  curriculum statements
+                </LI>
+              </UL>
+              <Typography $font={"heading-7"} $mb={12}>
+                <OakLink
+                  page={"webinar-index"}
+                  $display={"flex"}
+                  $alignItems={"center"}
                 >
-                  <CardLink
-                    page="blog-single"
-                    blogSlug="tbd-link-to-relevant-page"
-                  >
-                    <Heading tag={"h3"} $font={"heading-light-7"} $mb={16}>
-                      Resource
-                    </Heading>
-                    <P>How to design a unit of study</P>
-                  </CardLink>
-                </Card>
-              </GridArea>
-            </Grid>
-          </Box>
+                  Read more about our approach
+                  <Icon name={"chevron-right"} />
+                </OakLink>
```

Comment:
Double checked this with Hayley, and she's confirmed that this should link to [/blog/our-approach-to-curriculum](https://www.thenational.academy/blog/our-approach-to-curriculum)

Reply:
woops, all changed ✅